unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs

```diff
@@ -27,6 +33,8 @@ public class BIWCreatorController : BIWController
 
     private InputAction_Trigger.Triggered createLastSceneObjectDelegate;
 
+    private readonly Dictionary<string, GameObject> loadingGameObjectsDict = new Dictionary<string, GameObject>();
```

Comment:
```suggestion
    private readonly Dictionary<string, GameObject> loadingGameObjects = new Dictionary<string, GameObject>();
```

`Dict` can be redundant and unnecesary.